etc/bento.env

```diff
@@ -297,11 +297,11 @@ BENTOV2_GOHAN_ES_INTERNAL_PORT_1=9200
 BENTOV2_GOHAN_ES_INTERNAL_PORT_2=9300
 
 BENTOV2_GOHAN_ES_DISC_TYP=single-node
-BENTOV2_GOHAN_ES_MEM_LIM=8G
+BENTOV2_GOHAN_ES_MEM_LIM=4G
 BENTOV2_GOHAN_ES_CPUS=4
 
 # allocate a little bit less memory than what is stated in `mem_limit` ^^
-BENTOV2_GOHAN_ES_JAVA_OPTS="ES_JAVA_OPTS=-Xms8g -Xmx8g"
+BENTOV2_GOHAN_ES_JAVA_OPTS="ES_JAVA_OPTS=-Xms4g -Xmx4g"
```

Comment:
the issue here had more to do with the Xms4g/Xmx4g being too close to `BENTOV2_GOHAN_ES_MEM_LIM`

So i would set `BENTOV2_GOHAN_ES_MEM_LIM` back to 8G, and then `BENTOV2_GOHAN_ES_JAVA_OPTS` to `-Xms7800m -Xmx7800m` to set the heap size to 7.8 GB (a little bit below, like the comment says).

Reply:
keeping default container memory allocation as `4G` and set `ES_JAVA_OPTS`'s `Xms` and `Xmx` values to 3.8g